new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/TestActions.java

```diff
@@ -38,6 +44,37 @@ public static String createUniqueNamespace() {
         return Namespace.createUniqueNamespace();
     }
 
-    // etc...
+    // ------------------------ Docker image  -------------------------
+
+    public static boolean createMIIImage(
+        String baseImageName, 
+        String imageTag,
+        List<String> modelFiles,
+        List<String> modelVariableFiles,
+        List<String> modelArchiveFiles,
+        String wdtVersion,
+        String domainType) {
+  
+    	return 
+            new WebLogicImageTool()
+                .with(new WITParams()
+                     .baseName(baseImageName).tag(imageTag).modelFiles(modelFiles)
```

Comment:
this is just a nit pick, i prefer to put start a new line before every dot/period

Reply:
I assume baseImageName/Tag will itself eventually come from an 'ImageParams' type.

Reply:
@markxnelson it does make the logic clearer to start a new line before every dot.

Reply:
@tbarnes-us We could add an "ImageParams" if/when the need comes.